pkg/operator/configobservation/validation/validate.go

```diff
@@ -0,0 +1,9 @@
+package validation
+
+import "github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/configobservation/network"
+
+// Validate verifies whether the observed configuration can be used to make progress.
+func Validate(observedConfig map[string]interface{}) (errs []error) {
```

Comment:
this is correct, but extremely broad for abuse and drift.  How about a simple list of required keys?